runners/reference/job-server/pom.xml

```diff
@@ -122,5 +122,18 @@
       <artifactId>slf4j-jdk14</artifactId>
       <scope>test</scope>
     </dependency>
+
+    <!-- Maven Dependency Plugin expects this, but it should be included in hamcrest-all -->
```

Comment:
It actually is its just that the maven dependency plugin doesn't work well when one class is provided by two or more dependencies.

Reply:
ACK. I've made this a bit more terse just to explain why it's here at all.

Reply:
I'm gonna go ahead and say that the bug lies outside the dependency plugin.